tests/test-extensions/heisenberg-extension/src/main/java/org/mule/test/heisenberg/extension/HeisenbergScopes.java

```diff
@@ -84,6 +116,29 @@ public void neverFailsWrapper(@Optional Chain optionalProcessors,
     }
   }
 
+  public void asyncChainToObjectStore(@Expression(REQUIRED) Object latch,
+                                      Chain chain,
+                                      CompletionCallback<Void, Void> callback) {
+    final CountDownLatch countDownLatch = (CountDownLatch) latch;
+    final AtomicInteger counter = new AtomicInteger(0);
+    // scheduler.execute(() -> {
```

Comment:
remove?

Reply:
i will delete this test case alltogether. Ignore